src/pudl/package_data/eia860/column_maps/multifuel_retired.csv

```diff
@@ -0,0 +1,55 @@
+year_index,2004,2005,2006,2007,2008,2009,2010,2011,2012,2013,2014,2015,2016,2017,2018,2019,2020,2021
```

Comment:
Okay I think I was wrong. Looking at the `generation_fuel.csv` and `generators.csv` under `eia923/column_maps` I see that we do NOT have all of the eia923 years mapped for all tables. The `generators.csv` only goes back to 2008 and that's working fine, so I think in these column maps if there are years in which all values are blank because the spreadsheets don't exist for those years we can just not have columns for those years.

It also looks like for the column maps we aren't using `-1` sentinel values, and blanks are fine when a column doesn't exist in one year, but does in others.